src/adapters/peggedAssets/usd-coin/index.ts

```diff
@@ -1040,10 +1040,10 @@ const adapter: PeggedIssuanceAdapter = {
     unreleased: async () => ({}),
     ethereum: bridgedSupply("klaytn", 6, chainContracts.klaytn.bridgedFromETH),
   },
-  elrond: {
```

Comment:
id rather keep old name here since it will mess historical data otherwise

Reply:
Noted. Made a few tweaks to clean it up.

Reply:
Anything else needed to be tweaked? @0xngmi